airflow/models/dagrun.py

```diff
@@ -252,19 +252,22 @@ def refresh_from_db(self, session: Session = NEW_SESSION) -> None:
 
     @classmethod
     @provide_session
-    def active_runs_of_dags(cls, dag_ids=None, only_running=False, session=None) -> Dict[str, int]:
+    def active_runs_of_dags(
+        cls,
+        dag_ids: Optional[Iterable[str]] = None,
+        only_running: bool = False,
+        session: Session = NEW_SESSION,
+    ) -> Dict[str, int]:
         """Get the number of active dag runs for each dag."""
         query = session.query(cls.dag_id, func.count('*'))
         if dag_ids is not None:
-            # 'set' called to avoid duplicate dag_ids, but converted back to 'list'
-            # because SQLAlchemy doesn't accept a set here.
-            query = query.filter(cls.dag_id.in_(list(set(dag_ids))))
+            query = query.filter(cls.dag_id.in_(dag_ids))
```

Comment:
@uranusjr why this change?

Reply:
there are other places where this method is called, and they aren't all already set-like